recipe/meta.yaml

```
@@ -1,45 +1,55 @@
 {% set name = "portend" %}
-{% set version = "2.7.1" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "986ed9a278e64a87b5b5f4c21e61c25bebdce9919a92238d9c14c37a7416482b" %}
+{% set version = "3.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa9d40ab1f9e14bdb7d401f42210df35d017c9b97991baeb18568cedfb8c6489
 
 build:
-  noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
-  build:
-    - python >=3.6
+  host:
+    - python
     - pip
+    - setuptools >=77
     - setuptools_scm >=3.4.1
-
+    - wheel
   run:
-    - python >=3.6
+    - python
     - tempora >=1.8
 
 test:
+  source_files:
+    - test_portend.py
   imports:
     - portend
+  requires:
+    - python
+    - pip
+    - pytest
```

```suggestion
    - pytest  >= 6,!= 8.1.*
```
https://github.com/jaraco/portend/blob/v3.2.1/pyproject.toml#L36C9-L36C25